docker-compose-build.yml

```
@@ -136,30 +136,37 @@ services:
             - "8085:8085"
 
     elasticsearch:
-        image: acsdocker/elasticsearch:6.3.1-secured
+        image: bitergia/elasticsearch:6.1.0-secured
         command: /elasticsearch/bin/elasticsearch -E network.bind_host=0.0.0.0 -Ehttp.max_content_length=500mb
         networks:
             - default
         expose:
             - 9200
         ports:
           - "9200:9200"
+        volumes:
+          - /elasticsearch-index/:/elasticsearch/data
```

```suggestion
          - ~/elasticsearch-index/:/elasticsearch/data
```
We should avoid trying to mount volume on filesystem root.

---

Data can be lost in ES then. :thinking: